linkedlist.c

```diff
@@ -0,0 +1,53 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+struct Node{
+	int data;
+	struct node *next;

```

Comment:
Please upload error free code, this will give compilation error.
```suggestion
	struct Node *next;

```